### ISSUE TITLE
disk file viewer

### ISSUE THREAD

**Issue by rustyoz** [kierenj/0x10c-DevKit#162]
small plugin or application to view the current disk being used by the HMD plugin.

**Comment by kierenj** [comment]
Do you mean to view the contents of it?  In which format, a hex dump?

**Comment by rustyoz** [comment]
yes the contents. an option to view either either ascii from bytes or hex words. 

similar to issue #163 that i made. being able to read the contents of the disk either by using BIEF disks without compression or using a inbuilt view would allow for debugging of file system driver etc

the option of using uncompressed disks would also allow for standard text editors to manipulate disk content.

**Comment by rustyoz** [comment]
i decided to see if i could create one. 
turns out i could 

here it is https://github.com/bungao/BIEFconverter

**Comment by kierenj** [comment]
Cool, I see :)  Yes a disk viewer (in the hex view for instance) is a good idea, and possible.  It could show live changes too.  Not sure when it will be prioritised..
